src/IronPythonExtension/IronPythonExtension.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Dynamo.Extensions;
+
+namespace IronPythonExtension
+{
+    public class IronPythonExtension : IExtension
+    {
+        private const string PythonEvaluatorAssembly = "DSIronPython";
+
+        public void Dispose()
+        {
```

Comment:
Do you need to add a comment here as well to avoid the code smell?

Reply:
will do